health-services/project-factory/src/server/utils/processTrackUtils.ts

```
@@ -221,4 +221,153 @@ export function modifyProcessDetails(processDetailsArray: any[]) {
+    }
+
+    try {
+        const result = await executeQuery(query, values);
+        const resultantArray = result?.rows?.map((row:any) => ({
+            id : row.id,
+            campaignNumber: row.campaignnumber,
+            processName: row.processname,
+            status: row.status,
+            additionalDetails: row.additionaldetails,
+            createdBy: row.createdby,
+            createdTime: parseInt(row.createdtime),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)